store/types.go

```diff
@@ -55,20 +27,21 @@ type Deposit struct {
 type Output struct {
```

Comment:
Could we call this `OutputInfo`?

Reply:
I think the better way of handling this is to embed `plasma.Output`. Then the store types can be viewed as plasma types but with information spend information that the store has. We would just need to add RLPEnocde/RLPDecode funcs